app/controllers/order/goods_details.js

```diff
@@ -14,12 +14,16 @@ export default Ember.Controller.extend(cancelOrder, {
   qty: null,
   otherDetails: "",
   sortProperties: ["id"],
-  sortedGcRequests: Ember.computed("gcRequests.[]", function() {
-    return this.get("gcRequests");
-  }),
+  sortedGcRequests: Ember.computed(
+    "order.goodcityRequests",
+    "sortProperties",
+    function() {
+      return this.get("order.goodcityRequests");
```

Comment:
Is this sorted?? Because it doesn't look like it is

Reply:
Yes, it is sorted according to order of its creation.

Reply:
Because I see it's dependent on `sortProperties` but it is not used

Reply:
Got the issue, thanks for clarifying. Added sortProperties in sort computed property.